bin/bundler.rs

```diff
@@ -31,6 +31,13 @@ pub struct Opt {
 
     #[clap(long, default_value = "127.0.0.1:4337")]
     pub rpc_listen_address: String,
+
+    // execution client rpc endpoint
+    #[clap(long)]
+    pub network: SocketAddr,
```

Comment:
`pub eth_client_address: SocketAddr` would be better name IMO

Reply:
I copied the name from [typescript bundler](https://github.com/eth-infinitism/bundler/blob/main/packages/bundler/src/runBundler.ts#L88) and I agree that your name is better.